rave/src/main/java/com/uber/rave/Rave.java

```diff
@@ -69,6 +70,34 @@ public static synchronized Rave getInstance() {
     }
 
     /**
+     * Validate elements in a collection.
+     * @param collection the collection to validate
+     * @param <T> the type of the collection
+     * @throws RaveException if validation fails.
+     */
+    public <T> void validate(Collection<T> collection) throws RaveException {
+        for (T thing : collection) {
+            validate(thing);
+        }
+    }
+
+    /**
+     * Validate elements in a map.
+     * @param map the map to validate
+     * @param <K> key types
+     * @param <V> values types
+     * @throws RaveException if validation fails.
+     */
+    public <K,V> void validate(Map<K,V> map) throws RaveException {
```

Comment:
I'm not sure there's much value here since anything that's not annotated `@Validated` can't be validated. Getting a map key of type `String` from the network would just throw RaveException right?